### PR TITLE
Food class skeleton

### PR DESCRIPTION
**Pull request by Vinci-Hu**
Created food class and enum FoodCategory and FoodStorageLocation.

### CODE REVIEW

**Review by SimJJ96 — APPROVED** [review]
LGTM

File: src/main/java/seedu/duke/foods/FoodStorageLocation.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package seedu.duke.foods;
+
+/**
+ * Assume the fridge has these storage locations
+ * Reference: https://styledegree.sg/the-right-way-to-store-organize-food-in-fridge/
```
Comment (kwokyto) [inline src/main/java/seedu/duke/foods/FoodStorageLocation.java:5]: I like that you researched this! (:

**Review by kwokyto — APPROVED** [review]
LGTM

**Comment by joohwan58** [comment]
LGTM. Though can we change all instances of `Duke` to `FridgeFriend`?

**Pull request merged**